### DISCUSSION TITLE
[Eve] Matter support comes to Eve Energy, Eve Door & Window and Eve Motion on December 12

### DISCUSSION THREAD

**Discussion by lboue** [mfucci/node-matter#142] [Matter compatible devices / controllers]
[Matter support comes to Eve Energy, Eve Door & Window and Eve Motion on December 12](https://www.evehome.com/en/news/eve-welcomes-matter-with-free-firmware-updates-for-three-eve-products)

> Munich, Germany – November 3, 2022 – Eve Systems, the renowned innovator in the global smart home industry, today announced compatibility with Matter for three of its Thread-enabled smart home products. The latest models of the smart plug [Eve Energy ](https://www.evehome.com/eve-energy)(U.S. & Canada, Europe, U.K., and Australia), the contact sensor [Eve Door & Window](https://www.evehome.com/eve-door-window) and the motion sensor [Eve Motion](https://www.evehome.com/eve-motion) have completed Matter certification. Starting December 12, iPhone users can upgrade those devices for free in the Eve app to grant every family member local and direct access from their preferred smartphone or voice assistant.

**Comment by Apollon77** [comment]
Yes the problem is that this is all BLE or Thread ... Would jhave loved that one of the devices are WLAN :-(

**Reply by lboue** [reply]
> Yes the problem is that this is all BLE or Thread ... Would jhave loved that one of the devices are WLAN :-(

All you need is a Thread border router. This is how Threand and Matter works.
A WIFI connection consumes too much energy. Thread is much more efficient. Matter over WIFI will be used for bandwidth-intensive devices.

**Reply by Apollon77** [reply]
Yes I know ... but I do not have one ;-) (that was the main point) ... but I also have an Eve Energy already - and in fact you then again have an indirection, but all fine. That will all solve over time

**Comment by mfucci** [comment]
I think the border router makes those Matter devices accessible over WLAN: they are mapped to an IPv6 address and can be accessed transparently, the border router translating the protocol

**Reply by mfucci** [reply]
Seems like @lboue replied first!

**Reply by Apollon77** [reply]
;-)

**Comment by lboue** [comment]
Here we are:

### How to upgrade via the Early Access Program
[Getting started with Matter | evehome.com](https://www.evehome.com/en/meet-matter)

If you want to be among the first to leverage Matter and use your Eve devices from an additional platform, we invite you to join our Early Access Program. 

The Early Access Program is currently available for these Thread-enabled Eve devices:
- Eve Door & Window
- Eve Energy
- Eve Motion

**Apple**
Upgrading the firmware on an Eve device to Matter requires an **iPhone or iPad running iOS/iPadOS 16.2** (or later).

**Samsung**
Use the SmartThings app on your Android phone to control your Eve devices. Requires a SmartThings Hub v3.

**Reply by lboue** [reply]
[Theverge] [Matter is finally actually here](www.theverge.com/2022/12/12/23505097/matter-eve-firmware-update-available-now)

**Comment by Hedda** [comment]
FYI, EVE announced that their Matter firmware for Energy, Motion, Door & Window devices will soon be out of Beta (17th of April  2023);

https://www.evehome.com/en/upgrade-to-matter

* Eve Energy (Australia, Europe, U.K., U.S. & Canada)
  * https://www.evehome.com/en-us/identify-your-eve-accessory?product=eve-energy
* Eve Door & Window
  *  https://www.evehome.com/en-us/identify-your-eve-accessory?product=eve-door-window
* Eve Motion
  * https://www.evehome.com/en-us/identify-your-eve-accessory?product=eve-motion
